.github/workflows/sql-cli-test-and-build-workflow.yml

```diff
@@ -1,24 +1,33 @@
 name: SQL CLI Test and Build
 
-on: [pull_request, push]
+on:
+  pull_request:
+    types: [opened, reopened]
+  push:
+    branches-ignore:
+      - 'dependabot/**'
+    paths:
+      - 'sql-cli/**'
+      - '.github/workflows/sql-cli-test-and-build-workflow.yml'
 
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     defaults:
       run:
         working-directory: sql-cli
     strategy:
       matrix:
         python-version: [3.8]
+        opensearch-version: [ 2.2.1, latest ]
```

Comment:
why these two versions?

Reply:
There is no particular reason. Should I place there all released versions?

Reply:
maybe just latest?

Reply:
Fixed in 63c51377.